code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -18,15 +18,33 @@
 	return
 
 /obj/structure/largecrate/proc/unpack()
-	for(var/atom/movable/A in contents)
-		A.forceMove(loc)
+	var/turf/T = get_turf(src) // Get the turf the crate is on
+
+	var/list/temp_contents = contents.Copy() // Temporarily store the contents of the crate
```

Comment:
you're not altering the list here - do you need the copy?

Reply:
To the best of my current knowledge it would be possible to move it to around line 30.
I would prefer not to do this though so if this is required let me know otherwise I am marking this point as resolved.